packages/account/src/Sections/Security/Passwords/__tests__/deriv-email.spec.tsx

```diff
@@ -77,14 +77,14 @@ describe('DerivEmail', () => {
         let el_modal;
         await waitFor(() => {
             el_modal = screen.getByText('Change your login email');
```

Comment:
not sure why do we need here the variable in outer scope? can we just put getByText inside expect?

Reply:
We can. I thought this was more readable

Reply:
it is up to you, at least it looks like not habitual ,
I think we can reduce the quantity of lines if we just expect inside waitFor, how? I believe it will look like more clear